confd/warp/configReader.go

```diff
@@ -0,0 +1,107 @@
+package warp
+
+import (
+	"context"
+	"log"
+
+	"sort"
+
+	. "github.com/cloudogu/ces-confd/confd"
+	"github.com/coreos/etcd/client"
+	"github.com/pkg/errors"
+)
+
+type ConfigReader struct {
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ConfigReader should have comment or be unexported